corehq/pillows/domain.py

```diff
@@ -49,11 +49,11 @@ def calc_meta(self):
                                        "mapping": self.default_mapping})
 
     def change_transform(self, doc_dict):
-        doc_ret = copy.deepcopy(doc_dict)
         sub =  Subscription.objects.filter(
                 subscriber__domain=doc_dict['name'],
                 is_active=True)
         doc_dict['deployment'] = doc_dict.get('deployment', None) or {}
```

Comment:
I'd imagine the deep copy is done so `doc_dict` (the instance owned by the caller of this function) does not get modified. Would it be better to do this instead:
```py
doc_ret['deployment'] = doc_dict.get('deployment', None) or {}
```
Which avoids the side effect.